src/api/webhooks.js

```diff
@@ -27,8 +27,14 @@ export class WebhooksApi extends BaseApi {
             return Promise.reject(new Error('Must provide props.'));
         }
 
-        if (isTargetRequired && !props.target) {
-            return Promise.reject(new Error('Must provide a target.'));
+        if (isTargetRequired) {
+            if (!props.target) {
+                return Promise.reject(new Error('Must provide a target.'));
+            }
+
+            if (!props.target.startsWith('http://') || !props.target.startsWith('https://')) {
```

Comment:
This condition will always be true since it's `||`. A string can never start with both `http://` and `https://` :)

Should be `if (!props.target.startsWith('http://') && !props.target.startsWith('https://'))`